### ISSUE TITLE
Wrong Normals when importing Mesh

### ISSUE THREAD

**Issue by DeFYzz** [godotengine/godot#29904]
**Godot version:**
3.1.1

**OS/device including version:**
Windows 10 AMD RX 480

**Issue description:**
Wrong Lighting when importing mesh. Looks like Normals are flipped
Triangles which should be lit aren't lit and vice versa

**Steps to reproduce:**
- Import .obj models from pack: https://opengameart.org/content/nature-kit
- Add simple OmniLight and observe wrong lighting

**Comment by Llcoolsouder** [comment]
I just tried to reproduce this. It looks like the side of the model opposite the light is what is getting lit up. Is this what you're referring to?
![image](https://user-images.githubusercontent.com/15933766/59970492-ac6ada80-9535-11e9-9b7a-5a3020b2dee4.png)
![image](https://user-images.githubusercontent.com/15933766/59970499-d6240180-9535-11e9-898a-48a5a6f8f5ed.png)

**Comment by DeFYzz** [comment]
Yes that's exactly the issue

**Comment by Llcoolsouder** [comment]
@DeFYzz have you seen this same issue with other objs other than the one from this repo?

**Comment by Llcoolsouder** [comment]
I did a quick test to confirm that this was in fact a problem with the models and not an issue with the implementation of omnilight. I changed the color of the light to red so that it is easier to see its effects.

![image](https://user-images.githubusercontent.com/15933766/60846545-c818e680-a1ad-11e9-8491-6bd613949cef.png)

The monkey is just the standard mesh from Blender. Notice that the light behaves as expected, whereas both the flower and bamboo models have opposite lighting.

I think the easiest way to solve your problem would be to simply edit the models themselves. See the [Blender docs](https://docs.blender.org/manual/en/latest/modeling/meshes/editing/normals.html#flip-direction) for instructions on flipping the normals of the models. Hope that helps.
